src/Cli/dotnet/commands/dotnet-new/LocalizableStrings.resx

```diff
@@ -273,7 +273,7 @@
   </data>
   <data name="ProjectCapabilityConstraint_Restricted_NotRestored_CTA" xml:space="preserve">
     <value>Run 'dotnet restore {0}' to restore the project.</value>
-    <comment>do not translate 'dotnet restore {0}'</comment>
+    <comment>{Locked="dotnet restore {0}"}</comment>
```

Comment:
With new syntax, would it be automatic processing or still manual?
Otherwise not sure if it's worth replacing old text.

Reply:
I think the prior text should not have been used here and it was relying on the translator noticing the text whereas I think the Locked text actually affects the tooling on the loc side.

@cristianosuzuki77 any opinion on fixing old comments with the Locked format?

Reply:
we have been using the comments like this for quite a time, and it helped to avoid translation issues, so it seems the translators see them easily.

Reply:
Fair enough. I just don't know if the Locked format automatically removes them from the tooling so the exclusion is more automated than manual.